views/newsletter.ejs

```diff
@@ -993,7 +993,10 @@
                                                  style="border-radius: 5px;  border: none; -ms-interpolation-mode: bicubic; max-width: 100%;"
                                                  width="600">
                                         </td>
-                                    </tr><% if (post.featuredImageCaption) { %>
+                                    </tr>
+
+                                    <!-- I saw `null` as a text for `post.featuredImageCaption`, weird. -->
+                                    <% if (post.featuredImageCaption && post.featuredImageCaption !== 'null') { %>
```

Comment:
maybe we could try adding this check post removing whitespaces? Its possible that the value could be something like - ` null` or `null ` or ` null `, etc.

Reply:
Done.